### DISCUSSION TITLE
Herramientas facilitar el diseño web UI/UX

### DISCUSSION THREAD

**Discussion by RobertoMont31** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#300] [Ideas]
**Hola compañeros, siempre es bueno aprender desde las bases, pero una vez que se tiene dominado se vuelve un proceso aburrido y es cuando uno empieza a buscar herramientas o alternativas que le hagan a uno la chamba, me gustaría que pongamos aquí contribuciones de todo lo que algunos ya conozcan sobre UI/UX**

Dejo aquí mis primeras contribuciones: 

- https://webflow.com/  es un constructor de sitios web para ayudar a hacer más eficiente el proceso de diseñar botones, banners, landings, etc. 
- https://github.com/bradtraversy/design-resources-for-developers este es un repo que como su nombre lo dice es una LISTOTA pero LISTOTA de herramientas útiles para desarrolladores.

**Comment by OscarPedroza** [comment]
Muchas gracias por el aporte compañero
